src/org/opensolaris/opengrok/index/Indexer.java

```diff
@@ -726,23 +726,6 @@ public void prepareIndexer(RuntimeEnvironment env,
                     projects.add(p);
                 }
             }
-
-            // The projects should be sorted...
-            Collections.sort(projects, new Comparator<Project>() {
```

Comment:
We print them as sorted sets so this shouldn't be necessary after all. These changes were introduced in #1029 and before that PR we had had to sort the list of projects manually to display them. That seems like the original reason to me.